resource-service/build.gradle

```diff
@@ -0,0 +1,57 @@
+plugins {
```

Comment:
It is mostly same as server build.gradle. Could you move the similar parts to build.gradle at the root  project directory?